cd/update_versions.sh

```diff
@@ -27,6 +27,14 @@ source ./docker/accumulo2.env
 # JHUB_OPTIONS_SERVER_VERSION
 source ./docker/gaffer-jhub-options-server/get-version.sh
 
+# Update docker versions
+find . -type f -exec sed -i "s/GAFFER_VERSION=[0-9]\.[0-9]\.[0-9]/GAFFER_VERSION=${APP_VERSION}/g" {} +
```

Comment:
This is fine for now maybe could add an issue in for trying to simplify the Gaffer versioning to one place